src/Assets/ugui-mvvm/CommandBinding.cs

```diff
@@ -14,17 +15,20 @@ namespace uguimvvm
     public class CommandBinding : MonoBehaviour
     {
         [SerializeField]
-        Component _view;
+        [FormerlySerializedAs("_view")]
+        Component _target;
```

Comment:
Can we add tooltips to ensure people who are new to the mvvm worls don't get confused about where to associate their views and where to associate their viewmodels ?

Reply:
Since there is a custom editor that draws this label, it doesn't automatically show the `[Tooltip]`. I tried updating the editor from `EditorGUILayout.LabelField(property.displayName);` to `EditorGUILayout.LabelField(new GUIContent(property.displayName, property.tooltip));` but found that `property.tooltip` is always null, even if there is a `[Tooltip]` on the associated field. I found some [forum discussion](https://forum.unity.com/threads/serializedproperty-is-empty-with-tooltip-attribute.288159/) on this, but it sounds like Unity resolved the issue as *won't fix*. ☹

Reply:
Since there is a custom editor that draws this label, it doesn't automatically show the `[Tooltip]`. I tried updating the editor from `EditorGUILayout.LabelField(property.displayName);` to `EditorGUILayout.LabelField(new GUIContent(property.displayName, property.tooltip));` but found that `property.tooltip` is always null, even if there is a `[Tooltip]` on the associated field. I found some [forum discussion](https://forum.unity.com/threads/serializedproperty-is-empty-with-tooltip-attribute.288159/) on this, but it sounds like Unity resolved the issue as *won't fix*. ☹

Reply:
I recommend we address this in a separate PR since it might be a bigger change to the custom inspector.

Reply:
Update: we now display tooltips as requested.